application/controllers/bids.py

```diff
@@ -13,7 +14,7 @@
     'user_id': fields.Integer,
     'amount': fields.Float,
     'winner': fields.Boolean,
-    'created_at': fields.String
+    'created_at': fields.Integer,
```

Comment:
are you using Integer now because the date is logged in seconds instead of the traditional `datetime` syntax?

Reply:
We are actually using traditional timestamps. Which is typically utilized universally. We just haven't had the opportunity to work with them yet. A time stamp is the seconds from Jan 1st 1970. There is a library in JS that will run that timestamp and be more friendly to the FE to help them do what they need with a countdown.